salvo/salvo.py

```diff
@@ -13,14 +13,22 @@
 log = logging.getLogger()
 
 
-def setup_logging(loglevel=logging.DEBUG):
-  """Basic logging configuration """
+def setup_logging(loglevel: int=logging.DEBUG) -> None:
```

Comment:
(pre-existing) Since we are here and are improving the docstring, can we also add an "Args:" section and describe the loglevel argument? As discussed previously, we don't have to specify the type since we do have pytype, but we could say something about its meaning in the context of this function and Salvo as a whole.

Reply:
Updated.